osquery/tables/system/linux/dbus/methods/listunitsmethodhandler.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/tables/system/linux/dbus/methods/listunitsmethodhandler.h>
+
+namespace osquery {
+
+namespace {
+
+const std::size_t kMaxSystemdUnitCount{1000U};
```

Comment:
Is this max number of units defined by SystemD, or is this a limit chosen by us?

Reply:
This is a limit chosen by us, to make sure osquery will never loop forever enumerating things

Reply:
Ok, I'd support a code comment here to clarify that